_data/countries.yml

```diff
@@ -0,0 +1,17 @@
+# https://www.iban.com/country-codes
+
+ar: Argentina
+ca: Canada
+cn: China
+de: Germany
+es: Spain
+fr: France
+gb: United Kingdom
+in: India
+ja: Japan
```

Comment:
Japan's country code is `JP`. Meanwhile, `ja` is [the Japanese language code](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=ja).